gpii/node_modules/flowManager/test/configs/gpii.flowManager.tests.browserChannel.config.json5

```diff
@@ -18,6 +18,12 @@
                     "funcName": "gpii.tests.flowManager.browserChannel.reportPlatform"
                 },
                 "target": "{that deviceReporter platformReporter}.options.invokers.reportPlatform"
+            },
+            "browserChannel.tests.repositorySolutionsLoader": {
```

Comment:
Is this distribution still needed as now the SR remote fetch is controlled by an add-on config? The default behavior for all tests should be the same as having  SR remote fetch disabled.

Reply:
Right -- removed.  But, because these test definitions are shared with the acceptance tests in `tests/UntrustedBrowserChannelTests.js`, its config file needed a similar modifiction.